app/views/Dashboard/Country/index.tsx

```diff
@@ -394,98 +416,108 @@ function Country(props: Props) {
+                                        <div className={styles.regionalText}>
+                                            Regional 30%
+                                        </div>
+                                    </>
+                                )}
+                            />
+                        )}
+                        {literacyRate && (
+                            <TextOutput
+                                className={styles.countryTextOutput}
+                                valueContainerClassName={styles.valueText}
+                                labelContainerClassName={styles.labelText}
+                                hideLabelColon
+                                label="Literacy rate"
+                                value={(
+                                    <>
+                                        {`${literacyRate}%`}
+                                        <div className={styles.regionalText}>
+                                            Regional 30%
+                                        </div>
+                                    </>
+                                )}
+                            />
+                        )}
+                        {washAccessNational && (
+                            <TextOutput
+                                className={styles.countryTextOutput}
+                                valueContainerClassName={styles.valueText}
+                                labelContainerClassName={styles.labelText}
+                                hideLabelColon
+                                label="Access to basic washing facilities"
+                                value={(
+                                    <>
+                                        {`${washAccessNational}%`}
+                                        <div className={styles.regionalText}>
+                                            Regional 30%
+                                        </div>
+                                    </>
+                                )}
+                            />
+                        )}
+                        {countryResponse?.countryProfile.medicalStaff && (
+                            <TextOutput
+                                className={styles.countryTextOutput}
+                                valueContainerClassName={styles.valueText}
+                                labelContainerClassName={styles.labelText}
+                                hideLabelColon
+                                label="Doctors and nurses per 999 people"
+                                value={(
+                                    <>
+                                        {countryResponse?.countryProfile.medicalStaff
+                                            ? (countryResponse
+                                                ?.countryProfile.medicalStaff)?.toFixed(0)
+                                            : 'N/A'}
```

Comment:
If we've already checked this condition above, it'll never be false, which is why I think it's okay to just display the value without the 'N/A'